pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -21,7 +21,8 @@ This document describes the main concepts, requirements, and actions in {Product
 
 include::modules/ref_prerequisites.adoc[leveloffset=+1]
 
-== Overview of {ProductShortName}
+== Overview of {Prod
+uctShortName}
```

Comment:
@theashiot Pointing out this likely-unintentional line break.

Reply:
Thanks, not sure how I missed this! Fixed now